src/test/kotlin/org/rust/ide/refactoring/RenameTest.kt

```diff
@@ -83,6 +83,24 @@ class RenameTest : RsTestBase() {
         }
     """)
 
+    fun `test field initialization shorthand`() = doTest("spam", """
```

Comment:
Hmm, looks like it doesn't work with inplace renaming. Try the test in the IDE.

Reply:
Oh, yes, you are right. Seems that handleElementRename is not called on inplace renaming. What then I should do to handle this?

Reply:
I have fixed this issue during #4127. It has been merged recently so rebasing to latest master should solve it.

Reply:
Yes, rebase fixed that issue